sql/moz-fx-data-shared-prod/telemetry/forecasts_cache/view.sql

```diff
@@ -0,0 +1,51 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.forecasts_cache`
+AS
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.forecasts_cache_v1`
+UNION ALL
+SELECT
+  '' AS key,
+  CAST(ds AS TIMESTAMP) AS submission_date,
```

Comment:
Why timestamp rather than DATE here?

Reply:
I believe this is because looker requires it.

Reply:
If so, we should add a comment here to explain the mismatch.

Reply:
Right, Looker `date` support is a bit of an oddity right now. Once I'm able to stamp down just what the behavior is for different filter comparisons I'll open an issue with them to support `date` more robustly.